server/controllers/gift.controller.js

```diff
@@ -30,7 +29,7 @@ module.exports = {
     let giftParams = Object.assign({}, req.body, {event_id: req.params.id});
```

Comment:
But you don't reassign it.
https://google.github.io/styleguide/jsguide.html#features-local-variable-declarations